src/Todo.tsx

```diff
@@ -1,27 +1,19 @@
 import React from "react";
 
-class Todo extends React.Component<any> {
-	shouldComponentUpdate(prevProps: any) {
-	if(this.props != prevProps) {
-		return true;
-	}
-		return false;
-	}
-
-	handleOnClick() {
-		window.location.href = '/detail'
-	}
+interface TodoProps {
+    todo: {id: string; title: string;}
+}
 
-	render() {
+const Todo: React.VFC<TodoProps> = ({todo}) => {
+    const { title } = todo;
 
 	return (
 		<div>
-			<div onClick={this.handleOnClick}>
-			{this.props.todo.title}
-			</div>
+			<a href={'/detail'}>
```

Comment:
Anchor has a valid href attribute. It's a valid choice for navigation to other pages.